packages/web-components/src/spinner/spinner.stories.ts

```diff
@@ -0,0 +1,53 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import { SpinnerAppearance, SpinnerSize } from './spinner.options.js';
+import './define.js';
+
+type SpinnerStoryArgs = Args;
+type SpinnerStoryMeta = Meta<SpinnerStoryArgs>;
+
+const storyTemplate = html<SpinnerStoryArgs>`
+  <fluent-spinner appearance=${x => x.appearance} size=${x => x.size}></fluent-spinner>
+`;
+
+export default {
+  title: 'Web Components/Spinner',
```

Comment:
```suggestion
  title: 'Components/Spinner',
```
![image](https://user-images.githubusercontent.com/9615899/214926763-5909e9da-7cfc-4142-ae8e-163f60be5b47.png)